routes/membership.js

```diff
@@ -19,7 +19,20 @@ const {
 const router = express.Router();
 
 router.get('/', checkAdmin, async (req, res, next) => {
-	const season = await req.db.Season.getCurrent();
+	const selectedSeason = req.query.season;
+	let season;
+	if (selectedSeason == null) {
+		season = await req.db.Season.getCurrent();
+	}
+	else {
+		season = await req.db.Season.findOne({
+			where: {
+				id: req.query.season
+			}
+		});
+	}
+
+	const seasons = await req.db.Season.findAll();
 
 	if (!season) {
```

Comment:
This is a test for current season, and with this change it's not quite doing the right thing anymore.

This should probably read

```js
if (!selectedSeason && !season)
```